docs/settings/alert-action-settings.asciidoc

```diff
@@ -193,4 +193,11 @@ Specifies the default timeout for the all rule types tasks. The time is formatte
 For example, `20m`, `24h`, `7d`, `1w`. Default: `5m`.
 
 `xpack.alerting.cancelAlertsOnRuleTimeout`::
-Specifies whether to skip writing alerts and scheduling actions if rule execution is cancelled due to timeout. Default: `true`. This setting can be overridden by individual rule types.
\ No newline at end of file
+Specifies whether to skip writing alerts and scheduling actions if rule execution is cancelled due to timeout. Default: `true`. This setting can be overridden by individual rule types.
+
+`xpack.alerting.minimumScheduleInterval`::
+Specifies the minimum interval allowed for the all rules. This minimum is enforced for all rules created or updated after the introduction of this setting. The time is formatted as:
++
+`<count>[ms,s,m,h,d,w,M,Y]` 
```

Comment:
Updated in https://github.com/elastic/kibana/pull/125396/commits/688b74c62f02c492069ddd479c472872127e2996

Reply:
Ooops and again in https://github.com/elastic/kibana/pull/125396/commits/f165cd51095418eb22c6d41c9aaf7475be1f38bb because I didn't remove the larger values :)